src/views/Overview/EXV6Overview.tsx

```diff
@@ -172,6 +173,7 @@ export class EXV6Overview extends React.Component<{}, OverviewState> {
                                     offered_moderator_powers={user.offered_moderator_powers}
                                 />
                             )}
+                            <WarningSystemMessagesPane onlyOutstanding={false} />
```

Comment:
We need this on the Not EXV6 version too right? (... I think it's far past time we finish that "experiment up", maybe just add an option to have the stuff on the right or not - but that's something for a different day/pr :) )

Reply:
I'm thinking "if you have trouble with Warning Sytem Message, turn on this interface and take a look" ;)

Reply:
Oh okie dokie